index.rst

```diff
@@ -50,7 +50,696 @@
++------------------------+--------+-----------------------------------------------------------+
+| physical_sensor_id     | uint64 | NOT NULL REFERENCES PhysicalSensor (physical_sensor_id)   |
++------------------------+--------+-----------------------------------------------------------+
+| physical_filter_id     | uint64 | REFERENCES PhysicalFilter (physical_filter_id)            |
++------------------------+--------+-----------------------------------------------------------+
+| UNIQUE (first_visit, last_visit, camera_id, physical_sensor_id, physical_filter_id)         |
++------------------------+--------+-----------------------------------------------------------+
+
+Master calibration products are defined over a range of Visits from a given
+Camera, though a range of observation dates could be utilized instead.
+Calibration products may additionally be specialized for a particular
+PhysicalFilter, or may be appropriate for all PhysicalFilters by setting the
+``physical_filter_id`` field to ``NULL``.  Calibration products that are
+defined for individual sensors should use ``SensorCalibRange``.
+
+.. _cs_dataunit_joins:
+
+DataUnit Joins
+--------------
+
+The tables in this section represent many-to-many joins between DataUnits
+defined in the previous section that can be generated programmatically.  These
+join tables have no primary key (at least not as part of the common schema),
+and hence cannot be used to label Datasets.
+
++------------------+--------+---------------------------------------------------+
+| *CalibRangeJoin*                                                              |
++==================+========+===================================================+
+| calib_range_id   | uint64 | NOT NULL REFERENCES CalibRange (calib_range_id)   |
++------------------+--------+---------------------------------------------------+
+| visit_id         | uint64 | NOT NULL REFERENCES Visit (visit_id)              |
++------------------+--------+---------------------------------------------------+
+
++--------------------------+--------+-----------------------------------------------------------------+
+| *SensorCalibRangeJoin*                                                                              |
++==========================+========+=================================================================+
+| sensor_calib_range_id    | uint64 | NOT NULL REFERENCES SensorCalibRange (sensor_calib_range_id)    |
++--------------------------+--------+-----------------------------------------------------------------+
+| observed_sensor_id       | uint64 | NOT NULL REFERENCES ObservedSensor (observed_sensor_id)         |
++--------------------------+--------+-----------------------------------------------------------------+
+
+The above two tables define the joins between master calibration Datasets and
+the observations they should be used to calibrate.  These can be defined
+directly as views in on the DataUnit tables:
```

Comment:
Perhaps better to have them non-autogenerated instead? See above comment about contiguousness.